packages/manager/apps/pci-kubernetes/src/pages/reset/ResetCluster.page.tsx

```diff
@@ -94,7 +94,7 @@ export default function ResetClusterPage() {
   const [formState, setFormState] = useState({
     workerNodesPolicy: WORKER_NODE_POLICIES.DELETE,
     selectedVersion: getFormatedKubeVersion(kubernetesCluster?.version || ''),
-    privateNetworkId: kubernetesCluster?.privateNetworkId,
+    privateNetworkId: kubernetesCluster?.privateNetworkId ?? undefined,
```

Comment:
i know it is weird, but i need to send undefined to the server and not null

Reply:
@ppprevost 
Do you have any documentation on it?
In the API Schema, `privateNetworkId` is specified as below: 
```txt
privateNetworkId
string
OpenStack private network ID that the cluster will use. Optional, can only be set on cluster creation or reset. If unspecified, the cluster will use the public network.
```

Reply:
Yes it's optional so not null. 
Today i receive from the server null and send back null if i don't change the value which is normally ok but the server does not accept null but undefined. See with Nicolas Lacroux

Reply:
OK, but why not making the property optional in the object instead?
If the API property is optional, it should not be provided if undefined, it's clearer than sending `undefined value`: 
```suggestion
   ...( !!kubernetesCluster?.privateNetworkId && { privateNetworkId: kubernetesCluster?.privateNetworkId},
```